docs/bsp.md

```diff
@@ -792,6 +884,18 @@ trait TestReport {
 
   /** The total number of failed tests. */
   def failed: Int
+  
+  /** The total number of ignored tests. */
+  def ignored: Int
+
+  /** The total number of cancelled tests. */
+  def cancelled: Int
+
+  /** The total number of skipped tests. */
+  def skipped: Int
+
+  /** The total number of pending tests. */
```

Comment:
What does pending mean for a test?

Reply:
I agree it doesn't make much sense, I remove it

Reply:
I agree it doesn't make much sense, I remove it